tests/test_fgcmcal_hsc.py

```diff
@@ -105,7 +105,7 @@ def test_fgcmcalTasks(self):
         nGoodZp = 26
         nOkZp = 26
         nBadZp = 1206
-        nStdStars = 390
+        nStdStars = 389
```

Comment:
It's not clear to me why these magic numbers are changing.  Maybe insert a note in the commit message where they changed?